pages/getData.tsx

```diff
@@ -210,7 +296,13 @@ const GetData: NextPage = () => {
                 </div>
               </article>
               <pre style={{ wordBreak: 'break-all', whiteSpace: 'pre-wrap' }}>
-                {data}
+                {decodeData(data).map((element, index) => {
+                  if (element.permissionPresence) {
+                    return <p key={index}>{element.permissionName}</p>;
+                  } else {
```

Comment:
No need to add else and return

Juste remove the else and put a return

As the id is already returning